apps/mail-bridge/nitro.config.ts

```diff
@@ -26,6 +26,11 @@ const mailDomains: MailDomains = JSON.parse(process.env.MAIL_DOMAINS);
 
 // TODO: ensure limits are pulled from the billing module
 export default defineNitroConfig({
+  esbuild: {
+    options: {
+      target: 'EsNext'
+    }
+  },
```

Comment:
why are we targeting esnext?
what other impacts does this have?

Reply:
EsNext allows us to use top level await, I am using it for simpicity here, will change later if not needed

Reply:
i am keeping it as it makes lives easier